src/Model/Item.php

```diff
@@ -2064,6 +2063,10 @@ public static function getLanguageArray(string $body, int $count, int $uri_id =
 	 */
 	private static function splitByBlocks(string $body): array
 	{
+		if (class_exists('IntlChar')) {
```

Comment:
Doesn't this imply the `IntlChar` module is optional?

Reply:
Well, it isn't optional in the future. But without this check surely several installation would do some WSOD after install.